src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -77,6 +77,7 @@ bool BinaryCompactObject<UseWorldtube>::Object::is_excised() const {
 template <bool UseWorldtube>
```

Comment:
Re commit message: I'm not sure I'd call a ~150 line diff a "one line change" ;)